src/main/java/uk/gov/hmcts/reform/roleassignmentbatch/processors/EntityWrapperProcessor.java

```diff
@@ -20,31 +26,50 @@
-                                                   .id(requestUuid)
-                                                   .correlationId(UUID.randomUUID().toString())
-                                                   .clientId("ccd_migration")
-                                                   .authenticatedUserId("A fixed Authenticated User Id")
-                                                   .assignerId(ccdCaseUsers.getUserId())
-                                                   .requestType("CREATE")
-                                                   .status("APPROVED")
-                                                   .process("CCD")
-                                                   .replaceExisting(false)
-                                                   .roleAssignmentId(UUID.randomUUID())
-                                                   .reference(ccdCaseUsers.getCaseDataId().concat(ccdCaseUsers.getUserId()))
-                                                   .log(null)
-                                                   .created(LocalDateTime.now())
-                                                   .build();
+                                               .id(requestUuid)
+                                               .correlationId(UUID.randomUUID().toString())
+                                               .clientId("ccd_migration")
+                                               .authenticatedUserId("A fixed Authenticated User Id")
+                                               .assignerId(ccdCaseUsers.getUserId())
+                                               .requestType("CREATE")
+                                               .status("APPROVED")
+                                               .process("CCD")
+                                               .replaceExisting(false)
+                                               .roleAssignmentId(UUID.randomUUID())
+                                               .reference(ccdCaseUsers.getCaseDataId().concat(ccdCaseUsers.getUserId()))
+                                               .log(null)
+                                               .created(LocalDateTime.now())
+                                               .build();
+
+        HistoryEntity historyEntity = HistoryEntity.builder()
+                .id(requestUuid)
+                .requestEntity(RequestEntity.builder().id(requestUuid).build())
+                .actorId(ccdCaseUsers.getUserId())
+                .actorIdType("judge")
+                .roleType("judge")
+                .roleName("name")
+                .status("APPROVED")
+                .process("CCD")
+                .classification("org")
+                .grantType("admin")
+                .attributes(BooleanNode.getFalse())
+                .reference(ccdCaseUsers.getCaseDataId().concat(ccdCaseUsers.getUserId()))
+                .log(null)
+                .created(LocalDateTime.now())
+                .build();
```

Comment:
most of these values are dummy..can we please map the correct values as per mapping.